include/common_structs.h

```diff
@@ -1204,30 +1204,30 @@ typedef struct GameStatus {
     /* 0x0A8 */ s8 creditsViewportMode;
     /* 0x0A9 */ s8 unk_A9;
     /* 0x0AA */ s8 demoFlags;
-    /* 0x0AB */ u8 unk_AB;
-    /* 0x0AC */ s8 loadMenuState;
-    /* 0x0AD */ s8 menuCounter;
+    /* 0x0AB */ u8 usingStereoSound;
```

Comment:
If this is a bool I'd prefix with `is`...
```suggestion
    /* 0x0AB */ u8 isUsingStereoSound;
```
Or
```suggestion
    /* 0x0AB */ u8 isStereo;
```